ctfpad/templates/ctfpad/ctfs/detail_challenges.html

```diff
@@ -75,57 +88,46 @@
 
 
 <script>
-    function filterChallenges()
-    {
+    function filterChallenges() {
```

Comment:
Just realized, this now can go into `/js/challenges.js` but no that can be done in another PR